src/wp-includes/load.php

```diff
@@ -213,6 +213,7 @@ function wp_check_php_mysql_versions() {
  * @return string The current environment type.
  */
 function wp_get_environment_type() {
+	sleep( 13 );
```

Comment:
Why sleep?

Reply:
@mukeshpanchal27 That's me just testing to make sure I'm not about to commit broken tests. I'll be getting rid of all this weirdness soon. See https://github.com/WordPress/wordpress-develop/pull/4636/commits/acaabf27b64891e616601858584765281ebcc9a1